target/linux/mediatek/image/mt7622.mk

```diff
@@ -17,7 +19,8 @@ endef
 TARGET_DEVICES += MTK-LYNX-RFB1
 
 define Device/BPI-R64
-  DEVICE_TITLE := Banana Pi R64
+  DEVICE_VENDOR := Allnet
```

Comment:
While Allnet distributes the BananaPi boards, they are made by Banana-pi.org.
"BPI team is a RD team from sinovoip and Foxconn [...]" (see Company Bio on banana-pi.org)
Better use
```
  DEVICE_VENDOR := BananaPi
```
like for BPI-R2

Reply:
...or imaginary vendor `LeMaker` like for the sunxi BananaPi boards...
I don't mind either way, however, I'd be in favour of a consistent naming for that vendor.

Reply:
Thanks for the input. I used LeMaker now, since it the same for the other BananaPi boards.